drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/widget/table/utilities/ColumnUtilitiesTest.java

```diff
@@ -125,6 +132,148 @@ public void testCanAcceptOtherwiseWrongOperator() throws Exception {
         assertFalse(ColumnUtilities.canAcceptOtherwiseValues(column));
     }
 
+    @Test
```

Comment:
Might have been worth spitting your new tests into two classes and ```@Parameterized``` the first "batch", but an opinion rather than a requirement so take it or leave it.